From 304ce6e49d9a29b3b8a60343cf119b56c50f76a6 Mon Sep 17 00:00:00 2001
From: Sergey Senozhatsky <senozhatsky@chromium.org>
Date: Mon, 2 Mar 2020 17:52:05 +0900
Subject: [PATCH 3/3] CHROMIUM: config: Enable Intel camera drivers for volteer

This CL selects requied Intel camera drivers for volteer.

BUG=b:149068439, b:149068672
TEST=Sanity checked basic camera functions.

Change-Id: Ia1d7c520198ada8574a3ada9732b598f938d844b
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
---
 .../config/x86_64/chromeos-intel-pineview.flavour.config    | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config b/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
index 300a88de9917..f80a94ff07da 100644
--- a/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
+++ b/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
@@ -126,3 +126,9 @@ CONFIG_WILCO_EC_TELEMETRY=m
 CONFIG_X86_INTEL_LPSS=y
 # CONFIG_X86_MCE_AMD is not set
 CONFIG_X86_PKG_TEMP_THERMAL=y
+CONFIG_VIDEO_INTEL_IPU=m
+CONFIG_VIDEO_OV8856=m
+CONFIG_VIDEO_OV2740=m
+CONFIG_VIDEO_DW9768=m
+CONFIG_EEPROM_AT24=m
+CONFIG_VIDEO_INTEL_IPU6=y
-- 
2.17.1

